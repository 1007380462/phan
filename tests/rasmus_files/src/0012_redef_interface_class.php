<?php
interface test {
    function foo($arg1);
    function bar($arg2);
}

trait test {
    public function foo($arg1) { return $arg1; }
    protected function bar($arg2) { return $arg2; }
}

class test {
    public function foo($arg1) { return $arg1; }
    protected function bar($arg2) { return $arg2; }
}
