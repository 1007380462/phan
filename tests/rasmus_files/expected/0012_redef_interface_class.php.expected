%s:7 PhanRedefineClass Trait \test defined at %s:7 was previously defined as Interface \test at %s:2
%s:12 PhanRedefineClass Class \test defined at %s:12 was previously defined as Interface \test at %s:2
